### DISCUSSION TITLE
Release 1.3.4.0 LBT does not bind to Flysky IA6B anymore

### DISCUSSION THREAD

**Discussion by Quaxbruch** [pascallanger/DIY-Multiprotocol-TX-Module#997] [Q&A]
Hi Pascal. thank you very much on your efforts. I tried to update my IR4Xlite running under ETHOS 1.5.11 to latest release. I could not get my flysky receivers to bind anymore. It worked and works fine with the release 1.3.3.20.  Where is the error? Before the screen? kind regards Arthur

**Comment by pascallanger** [comment]
Have you tried: [v1.3.3.33](https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/releases/tag/v1.3.3.33)?
I haven't touched anything related to Flysky between v1.3.3.20 and v1.3.4.0...

**Comment by Quaxbruch** [comment]
Hi Pascal, I did now and it works pefectly fine. Kinde regards Arthur

The issues with 1.3.3.40 persist. It will not bind.

 
 

Gesendet: Dienstag, 30. Juli 2024 um 09:43 Uhr
Von: "pascallanger" ***@***.***>
An: "pascallanger/DIY-Multiprotocol-TX-Module" ***@***.***>
Cc: "Quaxbruch" ***@***.***>, "Author" ***@***.***>
Betreff: Re: [pascallanger/DIY-Multiprotocol-TX-Module] Release 1.3.4.0 LBT does not bind to Flysky IA6B anymore (Discussion #997)


 

Have you tried: v1.3.3.33?
I haven't touched anything related to Flysky between v1.3.3.20 and v1.3.4.0...

**Reply by Quaxbruch** [reply]
Not sure what happened, but the Tandem 18s info screen now thros a LUA error and deos not show the TX module (int and ext ) any more. instead ist writes between the line for date and stickmode a line reading: Error 21:40:46 (which is the boot time) Lua widget key " not found. 
don't know which Lua generates that error.

**Reply by christophe-heidet** [reply]
I had the same problem with a FrSky X14 on ETHOS 1.6.0 and a FlySky FS-IA6B receiver.
I solved by flashing the IRX4Lite with the 1.3.4.0 AIR firmware.

The FS-IA6B I bought on Aliexpress appears to be a non EU/LBT version

**Comment by Quaxbruch** [comment]
found it on github: https://github.com/FrSkyRC/ETHOS-Feedback-Community/issues/4079   what a mess. case closed
